arch/arm64/boot/dts/qcom/sdm660-xiaomi-lavender-common.dtsi

```diff
@@ -140,10 +164,70 @@
 	};
 };
 
+&gpucc {
+	status = "okay";
+};
+
 &lpass_smmu {
 	status = "okay";
 };
 
+&kgsl_smmu {
```

Comment:
kgsl_smmu should go between gpucc and lpass (alphabetical order)

Reply:
mistake in my commit I guess?

Reply:
OK, I have fixed it, force pushed my wip branch as well as target branch qcom-sdm660-6.8.y, but can't update this MR :(
```
$ git push -f barni2000 HEAD:lavender_split
ERROR: Permission to barni2000/linux.git denied to minlexx.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```
since you're now member of this org, can you maybe recreate MR but using branch in this repo?